### DISCUSSION TITLE
Wails Desktop App UI Rendering Issue on Raspbian OS

### DISCUSSION THREAD

**Discussion by core-sciverse** [wailsapp/wails#3084] [General]
I'm currently facing an issue with a Wails desktop application that I've been developing. The application works seamlessly on standard desktop environments, but I'm encountering difficulties with UI rendering when trying to export and run it on Raspbian OS (Raspberry Pi).

**Comment by ansxuman** [comment]
In main.go ,inside 	`err := wails.Run(&options.App{ })` add the below field

```Linux: &linux.Options{
			WebviewGpuPolicy: linux.WebviewGpuPolicy(2)}
```

**Comment by core-sciverse** [comment]
Thank you.
